pkg/discovery/processor/gitops_config_processor.go

```diff
@@ -0,0 +1,30 @@
+package processor
```

Comment:
I am not sure if we really need this processor.

Reply:
Removed the processor.